### PR TITLE
Refactored prim utils into _prims_utils folder

### PR DESCRIPTION
**Pull request by Chillee**
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #81093
* #81092
* __->__ #81088

### CODE REVIEW

**Comment by Chillee** [comment]
Ugh this doesn't actually work, since `wrappers` calls a lot of prim ops.

**Comment by Chillee** [comment]
For the most part, this is a mechanical change, moving `_prims/utils.py` and `_prims/wrappers.py` to `_prims_common` and updating imports.

2 non-trivial changes:

1. I moved `get_prim_fake_mode` and `TensorMeta` into `_prims/__init__.py`. 
2. I updated the `prims` calls in `_prims_common/wrappers.py` to call torch ops instead (with the exception of `copy_to`, which I changed to a lazy import).

File: torch/_prims_common/__init__.py [hunk 1]
```diff
@@ -241,140 +160,6 @@ def is_contiguous(a: TensorLikeType) -> bool:
     return True
 
 
-# This function is equivalent to compute_channels_last_contiguous_2d() in TensorImpl.cpp
-def is_channels_last_contiguous_2d(a: Tensor) -> bool:
```
Comment (ngimel) [inline torch/_prims_common/__init__.py:245]: where did all these functions go?

Comment (Chillee) [reply torch/_prims_common/__init__.py:245]: hrmm, I suspect a recent merge failure.

Comment (ngimel) [reply torch/_prims_common/__init__.py:245]: oh ok, I think constant_pad_nd was reverted

**Review by ngimel — APPROVED** [review]

File: torch/_prims_common/__init__.py (hunk 1 above)
Comment (Chillee) [reply torch/_prims_common/__init__.py]: Fixed the issue.

**Comment by Chillee** [comment]
@jeanschmidt what's the failure?

**Comment by kit1980** [comment]
> what's the failure?

See D37957414

**Pull request closed without merging**